cloud-account/update-your-email-address.md

```diff
@@ -1,18 +1,21 @@
 ---
 mapped_pages:
   - https://www.elastic.co/guide/en/cloud/current/ec-update-email-address.html
+applies:
+  serverless: all
+  hosted: all
 ---
 
 # Update your email address [ec-update-email-address]
 
 Each Elasticsearch Service account has a primary email associated with it. By default, the primary email address is used to sign up for Elasticsearch Service and to log in. If needed, you can change this primary email address:
```

Comment:
Consider splitting this into two child procedures with their own headings for scannability. not critical to do so

Reply:
I don't follow what you are talking about here, sorry!
What should be split?